params/safety_config.yaml

```diff
@@ -0,0 +1,11 @@
+safety_bonds:
```

Comment:
Problem with that is that it requires this launch file to be loaded before rosparam stuff from other launch files, which is why I didn't do that